src/orquestra/sdk/_base/_logs/_models.py

```diff
@@ -0,0 +1,46 @@
+################################################################################
+# © Copyright 2023 Zapata Computing Inc.
+################################################################################
+"""
+Models for managing logs, used internally.
+"""
+from enum import Enum
+from pathlib import Path
+from typing import Dict, List, Sequence
+
+
+class LogStream(Enum):
```

Comment:
`LogStream` is confusing for me. Anytime I see `stream` I expect something I can lazily read data from. This is not it, it's an enum.

```suggestion
class LogStreamType(Enum):
```
or
```suggestion
class LogStreamKind(Enum):
```

Reply:
Changed